pkg/sql/sem/builtins/builtins.go

```diff
@@ -3721,9 +3721,50 @@ value if you rely on the HLC for accuracy.`,
 				"charge for each edit operation. Maximum input length is 255 characters.",
 			Volatility: tree.VolatilityImmutable,
 		}),
-	"levenshtein_less_equal": makeBuiltin(tree.FunctionProperties{UnsupportedWithIssue: 56820, Category: categoryFuzzyStringMatching}),
-	"metaphone":              makeBuiltin(tree.FunctionProperties{UnsupportedWithIssue: 56820, Category: categoryFuzzyStringMatching}),
-	"dmetaphone_alt":         makeBuiltin(tree.FunctionProperties{UnsupportedWithIssue: 56820, Category: categoryFuzzyStringMatching}),
+	"levenshtein_less_equal": makeBuiltin(defProps(),
+		tree.Overload{
+			Types:      tree.ArgTypes{{"source", types.String}, {"target", types.String}, {"max_d", types.Int}},
+			ReturnType: tree.FixedReturnType(types.Int),
+			Fn: func(evalCtx *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
+				s, t := string(tree.MustBeDString(args[0])), string(tree.MustBeDString(args[1]))
+				m := int(tree.MustBeDInt(args[2]))
+				const maxLen = 255
```

Comment:
can you add a comment of why is there a 255 char limit?
seems like it's a) what postgres does (which justifies this) and b) because it builds a N*M matrix